src/main/menu/application-menu-items.injectable.ts

```
@@ -69,7 +69,8 @@ const applicationMenuItemsInjectable = getInjectable({
             label: `About ${productName}`,
             id: "about",
             click(menuItem: MenuItem, browserWindow: BrowserWindow) {
-              showAbout(browserWindow);
+              showAbout(browserWindow)
+                .catch(error => logger.error(`[MENU]: Failed to show Lens About view`, { error }));
```

This logic to catch, log and suppress errors is duplicated in many places.